### ISSUE TITLE
Print multiple files at once?

### ISSUE THREAD

**Issue by Polm90** [katzer/cordova-plugin-printer#227]
Is it possibile to print multiple files at once? For example, printing 2 or 3 PDF files with only one call to print function?

**Comment by vishwas097** [comment]
@Polm90 Did you find a way to do that?

**Comment by Polm90** [comment]
No

**Comment by vishwas097** [comment]
@Polm90 Do you have any idea on printing via html content? Like if i need to print different html contents in two different pages at a time. Is that possible to do that?

**Comment by Polm90** [comment]
You can build a single html document with all the content you need. To be sure to break page as needed, I suggest you to use the css page-break rules

**Comment by vishwas097** [comment]
@Polm90  Nice idea and thank you very much!

**Comment by Phoenix-Alpha** [comment]
Anyone found solution to print multiple files at once?

**Comment by alexcodelab** [comment]
@Phoenix-Alpha Yes you need append after a photo or doc this code `<p style="page-break-after : always"></p>`.
Then call the print function `cordova.plugins.printer.print($('#divwithalldocs').html());` and you get multiple prints.
Thanks to @Polm90 very nice idea!
